### DISCUSSION TITLE
Update firmware failed with command AT1U=6502

### DISCUSSION THREAD

**Discussion by andromeda2911** [bozimmerman/Zimodem#106] [Q&A]
How can i update firmware to 3.7.1 ?
this command failed (at&u=6502) => updating to 3.7.1 wait for modem restart...ERROR
i have tested at&u=3.7.1 it don' work.
i use 9600 bauds.
i can connect to telnet on my pi zero, configuration wifi is ok
i use ESP-01 with 1024k
Thanks in advance.

**Comment by mkubiak402** [comment]
yeah don't work for me and the AT&H is all so broken.
